packages/contracts/test/contracts/OVM/execution/OVM_ExecutionManager/ovmCREATE.spec.ts

```diff
@@ -24,14 +24,21 @@ const DUMMY_REVERT_DATA =
   '0xdeadbeef1e5420deadbeef1e5420deadbeef1e5420deadbeef1e5420deadbeef1e5420'
 
 const NON_WHITELISTED_DEPLOYER = '0x1234123412341234123412341234123412341234'
-const NON_WHITELISTED_DEPLOYER_KEY =
-  '0x0000000000000000000000001234123412341234123412341234123412341234'
+const NON_WHITELISTED_DEPLOYER_KEY = ethers.utils.keccak256(
```

Comment:
Assuming that this can't be easily done because it lives in smock right now, but this should really be a call to a utility function.  Took me a sec to realize this was here to handle the new solidity storage layout.

Reply:
Yeah, very true!